extensions/amp-story/1.0/amp-story.css

```diff
@@ -380,18 +380,13 @@ amp-story-grid-layer .i-amphtml-embedded-component::after {
   padding: 0 !important;
 }
 
-.i-amphtml-story-grid-template-fill > :not(:first-child) {
-  display: none !important;
-}
-
-.i-amphtml-story-logo {
```

Comment:
Since we are already deleting this one, and the only other reference I can find is in `amp-story-desktop-panels.css`, can we also delete that one as well please?